src/app/components/ReadTime/index.tsx

```diff
@@ -0,0 +1,30 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { use } from 'react';
+import isLive from '#app/lib/utilities/isLive';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import styles from './index.styles';
+
+type ReadTimeProps = {
+  readTime: number;
```

Comment:
Should this not be optional in case there is no read time?
```suggestion
  readTime?: number;
```